pyproject.toml

```diff
@@ -75,3 +75,10 @@ log_cli_level = "DEBUG"
 [tool.mypy]
 no_strict_optional = true
 ignore_missing_imports = true
+
+[tool.codespell]
+# Ref: https://github.com/codespell-project/codespell#using-a-config-file
+skip = '.git*'
+check-hidden = true
+ignore-regex = '(^\s*"image/\S+": ".*|ND)'
```

Comment:
These are here to suppress false positives? We will see how long this gets 😅.

Reply:
I fixed `ND` to be a complete word now, that is what tripped CI. For the ipynb included images -- quite a reliable pattern , wouldn't expect problems.